### ISSUE TITLE
CRAN release

### ISSUE THREAD

**Issue by daroczig** [daattali/shinymixpanel#2]
Hey,

Thanks a lot for the great package! I just wanted to note that the package doesn't seem to be on CRAN despite it being mentioned in the README.

Best,
Gergely

**Comment by daattali** [comment]
That's true! I have several packages that are due to get on CRAN... I just need to finish their documentation. I'm really sorry for falling so behind on this.

**Comment by daroczig** [comment]
No rush, I'm good using the GH version with pinned commit hash, just wanted to note the discrepancy between the `README.md` and reality :)
